Library/Formula/mpd.rb

```diff
@@ -61,6 +61,7 @@ class Mpd < Formula
   depends_on "opus" => :optional        # Opus support
   depends_on "libvorbis" => :optional
   depends_on "libnfs" => :optional
+  depends_on "mad" => :optional
```

Comment:
Shouldn’t that be `:recommended` to keep the current behavior? `:optional` means it’s not included by default while `:recommended` means it’s always included unless you use `--without-mad`?